### ISSUE TITLE
[Feature Request] - Option to merge graphs, grafana style

### ISSUE THREAD

**Issue by benjmd** [henrywhitaker3/Speedtest-Tracker#199]
Hiya, any possibility of merging the graphs so that up/down/ping are on the same graph, where clicking on the line/item selects or deselects from view?

**Comment by henrywhitaker3** [comment]
I'll have a look into it

**Comment by henrywhitaker3** [comment]
Yeah will work on a toggle that merges dl/ul/ping graphs. As it stands you can hide one of the lines from view by clicking in the legend of the graph

**Comment by benjmd** [comment]
Terrific! thanks so much
